Pinta.Core/Classes/Re-editable/Text/TextLayout.cs

```diff
@@ -35,7 +35,7 @@ namespace Pinta.Core
 {
     public class TextLayout
     {
-        private TextEngine engine;
+        private TextEngine engine = null!; // NRT - Not sure how this is set, but all callers assume it is not-null
```

Comment:
Yeah, this is also set through the text tool and should be non-null. It probably needs some refactoring to make the behaviour more clear